src/runtime/policy/tests/policy-test.ts

```diff
@@ -687,4 +695,148 @@ policy MyPolicy {
       assert.isFalse('Person' in maxReadSchemas);
       assert.deepEqual(maxReadSchemas['Name'], expectedSchemas['Name']);
     });
+
+  it('returns max read type according to multiple policies', async () => {
```

Comment:
this test is really long and hard to follow, is it possible to factor the initialization part out into a helper method, and have a separate small test methods?

Reply:
I have split it up into three tests. PTAL.